pkg/sentry/fs/proc/sys_net.go

```diff
@@ -280,11 +280,120 @@ func (p *proc) newSysNetCore(ctx context.Context, msrc *fs.MountSource, s inet.S
 	return newProcInode(ctx, d, msrc, fs.SpecialDirectory, nil)
 }
 
+// ipForwarding implements fs.InodeOperations.
+// ipForwarding is used to enable/disable packet forwarding of netstack
+// and cache user input to "/proc/sys/net/ipv4/ip_forward".
+//
+// +stateify savable
+type ipForwarding struct {
+	stack inet.Stack `state:"wait"`
+	fsutil.SimpleFileInode
+
+	// mu protects enabled
+	mu sync.Mutex
+
+	// Linux stores these as an integer,
+	// so if you write "2" into ip_forward, you should get "2" back.
```

Comment:
We don't implement this behavior for the other options and applications don't seem to rely on it. Better to use a bool for consistency?

Reply:
It was originally a bool and I suggested it be an int32 if it wasn't an issue to just make it compatible with Linux but yeah consistency with other options is probably more important.

Reply:
fixed